src/lib/translators/mongo.ts

```diff
@@ -222,6 +223,43 @@ function transformTop(step: TopStep): Array<MongoStep> {
   ];
 }
 
+/** transform an 'unpivot' step into corresponding mongo steps */
+function transformUnpivot(step: UnpivotStep): Array<MongoStep> {
+  let mongoPipeline: Array<MongoStep> = [];
+  let projectCols: PropMap<string> = {};
+  let objectToArray: PropMap<string> = {};
+
+  // Prepare the projectCols to be included in Mongo $project steps
+  for (const col of step.keep) {
```

Comment:
Since we now depend on lodash, I suggest the following:

```typescript
import _ from 'lodash';

const projectCols: PropMap<string> = _.fromPairs(step.keep.map(col => [col, `$${col}`]));
```